pylint/checkers/refactoring.py

```diff
@@ -273,6 +273,11 @@ class RefactoringChecker(checkers.BaseTokenChecker):
             "consider using the list, dict or set constructor. "
             "It is faster and simpler.",
         ),
+        "R1722": (
+            "Consider using sys.exit()",
+            "consider-using-sys-exit",
```

Comment:
I don't know if there are strict conventions about this, but I think the name of the warning should be the diagnosis, not the suggested course of action. In this case, that would mean changing the name of the warning to `interactive-exit` or something like that.

Reply:
That is an interesting point. We have a couple of checks named in this way, which is probably why @sushobhit27 preferred this format. It would be great to standardize the naming somehow.

Reply:
One way to look at it is that the name of the check should complete the sentence "This is a check for ...". Another way to look at it is that adding `# pylint: disable=...` is the programmer's way of saying "I am explictly opting in to using ...". There are some existing examples that fit this format, like `broad-except` and `bare-except`.

It would be possible to standardize these in a backwards-compatible way by adding aliases or whatever, but that discussion is obviously outside the scope of this PR.